### ISSUE TITLE
Release twisted names 0.3.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5081]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#1731|
|Type|task|
|Created|2006-05-21 21:53:56Z|

It is time.

<details><summary>Searchable metadata</summary>

```
trac-id__1731 1731
type__task task
reporter__radix radix
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__names names
keywords__ 
time__1148248436000000 1148248436000000
changetime__1148250789000000 1148250789000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set status to `closed`|
|-|-|

(In [16900]) News and Versioning from Twisted Names 0.3.0.

Author: Release automation, radix
Reviewers: radix, exarkun
Fixes [#5081](https://github.com/twisted/twisted/issues/5081)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
